nebu/models/binder.py

```diff
@@ -136,7 +137,8 @@ def func(reference, resource):
             module_id_pattern = re.compile(r'\/(m\d{5})(@\d+[.]\d+([.]\d+)?)?')
             module_id_match = module_id_pattern.search(reference.uri)
 
-            if resource:
+            # FIXME: Probably safe to remove resource part
+            if resource:  # pragma: no cover
```

Comment:
More details: I am less sure about this one. It is eventually used in Document.__init__. In the [coverage report](https://philschatz.com/enki-coverage/coverage/_workspace_enki_venv_lib_python3_8_site-packages_nebu_models_document_py.html#t165), however, it looks like none of the resources ever resolve to a local path and [always returns None](https://philschatz.com/enki-coverage/coverage/_workspace_enki_venv_lib_python3_8_site-packages_nebu_models_binder_py.html#t139). Consequently, the entirety of resource handling, discovery, and resolution might be completely redundant.